hw/ip/aon_timer/doc/_index.md

```diff
@@ -20,8 +20,10 @@ See that document for an overview of how it is integrated into the top level sys
 
 The always-on wakeup timer operation is straightforward.
 A count starts at 0 and slowly ticks upwards (one tick every `N + 1` clock cycles, where `N` is the pre-scaler value).
-When it reaches / exceeds the wake count, a level wakeup signal is sent to the power manager and a level IRQ is sent to the processor.
-This wakeup signal stays high until it is explicitly acknowledged by software (software must write zero to the {{<regref "WKUP_CAUSE">}} register to clear it).
+When it reaches / exceeds the wake threshold, a level wakeup signal is sent to the power manager and a level IRQ is sent to the processor.
+This wakeup signal stays high until it is explicitly acknowledged by software.
+Software must write 0 to the {{<regref "WKUP_CAUSE">}} register to clear the wakeup and 1 to {{<regref "INTR_STATE.wkup_timer_expired">}} to clear the interrupt.
```

Comment:
Nit: Maybe "... and then 1 to ..." to make it clear that it has to happen in this order (to stop the interrupt re-triggering)?

Reply:
They're actually not related. The wakeup signal is going to the power manager, the interrupt is going to the Ibex external IRQ. I'll rewrite into two sentences to make it clear they're separate.

Reply:
Oh! That makes sense, thanks.